Update SRG-OS-000433-GPOS-00192 for RHEL 9 STIG

jan-cerny opened the pull request:
#### Description:
Add OCIL and fixtext to rules
Unify GRUB 2 remediation
Create fixtext Jinja macros from GRUB argument rules

#### Rationale:
STIG RHEL 9

Mab879 commented on linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/rule.yml:
```diff
@@ -61,3 +61,21 @@ ocil: |-
+
+    Then, update the GRUB configuration using the following command:
+
+    $ sudo grub2-mkconfig -o "{{{ grub2_boot_path }}}"/grub*.cfg
```
Shouldn't this use `gubby` for RHEL9?

jan-cerny commented:
I have changed the PR to use grubby in RHEL 9.

jan-cerny commented:
> I have changed the PR to use grubby in RHEL 9.

I think that we will need to have some Jinja macro fixtext_grub_ .... and use it at multiple places, there are multiple rules that need change, for example in https://github.com/ComplianceAsCode/content/pull/8616

Mab879 reviewed (changes requested):
One minor nit.

jan-cerny commented:
Now I have add Jinja macros for GRUB 2 fixtext and remediations and also updated the rule accordingly, then I have removed items related to 32 bits systems on RHEL and Fedora products.

Mab879 commented:
I am waving SSGTS as the tests pass locally.

matejak commented on linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/ansible/shared.yml:
```diff
@@ -4,6 +4,9 @@
 # disruption = low
 
+{{# products that are available also in a 32 bits form #}}
+{{% if "rhel" not in product and product != "fedora" %}}
```
Late to the party, but perhaps somebody notices: Why not to introduce an explanatory variable `product_has_32bit_form`? It's better to to pair products with properties, and then to work with those properties rather than with lists of products, mentioning the property as a comment.
